openedx/core/djangoapps/user_api/views.py

```diff
@@ -162,6 +163,7 @@ class RegistrationView(APIView):
         "gender",
         "year_of_birth",
         "level_of_education",
+        "dropdown",
```

Comment:
I don't like that you're calling this `'dropdown'` everywhere. It's very generic. Why not `custom_dropdown` or `custom_reg_field`? Why does it have to be a dropdown item?